app/policies/authorizer.rb

```diff
@@ -46,7 +46,7 @@ def admin?
     def administrative_access_to?(resource:, role_name: :single_resource_admin)
       # The implementation details of rolify are such that we can't
       # quite combine these functions.
-      return true if has_any_role?(*ANY_ADMIN_ROLES)
+      return true if has_any_role?(*ANY_ADMIN_ROLES) || moderator?
```

Comment:
This change was NOT necessary for user-suspension to work locally.

However, in Cypress, I kept getting a Puma `Unauthorized` error when the test performed the user-suspension. Upon investigating, the error led me to this file.

I am unsure if the addition of `|| moderator?` will confer unintended authority to the `moderator` role?

@jeremyf